FEXCore/Source/Interface/IR/IR.json

```diff
@@ -895,6 +895,10 @@
           "Size == FEXCore::IR::OpSize::i32Bit || Size == FEXCore::IR::OpSize::i64Bit"
         ]
       },
+      "GPR = AddNZCV u8:$Size, GPR:$Src1, GPR:$Src2": {
```

Comment:
This should have an EmitValidation section like the IR op above it so the IR validation can also know that it only supports 32bit and 64bit operation.

Reply:
Probably also use `OpSize:#Size` instead of `u8:$Size`